meta-balena-reference/layers/meta-balena-bsp/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,35 @@
+IMAGE_FSTYPES_append_qemux86 = " resinos-img"
+IMAGE_FSTYPES_append_qemux86-64 = " resinos-img"
+
```

Comment:
Isn't there a qemuall family override? I would think the settings above apply also for qemuarm and qemuarm64.
```
IMAGE_FSTYPES_append_qemuall = " resinos-img"
```